packages/editor/src/client/blocks/mapLeafletBlock/mapLeafletBlock.tsx

```diff
@@ -43,6 +36,12 @@ export interface MapLeafletBlockProps extends BlockProps<MapLeafletBlockValue> {
 
 export function MapLeafletBlock({value, onChange, disabled}: BlockProps<MapLeafletBlockValue>) {
   const {zoom, centerLat, centerLng, caption} = value
+  const MarkerIcon = Leaflet.icon({
```

Comment:
Variable names start with lowercase letters. More information about naming and variables in JS here https://javascript.info/variables